src/Element.js

```diff
@@ -74,7 +73,7 @@ class Element extends React.Component {
             ? palette[colorProp]
             : colorProp;
 
-        const fontStyle = typoStack[typography];
+        const fontStyle = themeTypography[typography];
```

Comment:
Since you're defaulting `muiTheme` to be an empty object, the later destructing of `palette` and `themeTypography` on line 66 would seem to have the potential to result in those variables not having values either.  Or, am I missing out on the destructing syntax or something in ES2016 or ES2017? 

If this is the case, then it might make sense to default `muiTheme` to an object containing a default for `palette` and `typography` as well?

Reply:
You are correct, it should be defended.

This brings up a point that Jason has made (I think I agree with him) which is that the cyverse theme should be required and other users like jetstream should override these values to modify. Like MUI does the CY-UI components should use values from the theme and not internally thus blow up with a message if not present.

Because of this we should probably throw an error message explaining this requirement.  Since all components use Element as a base this would be a good place to catch that.

Does that seem reasonable?

Reply:
Do you want the components to offer a warning, or outright _not_ render if a theme is missing?

Reply:
When I hear "override", that makes me think that, _by default_, the CyVerse theme is used. 

If that is so, then the absence of a theme is unexpected - _thus_ an error.

If you have to provide a theme because there is no default ... then you would likely want to "fail fast" so that the person using the component library is given clear feedback. 

I don't have strong opinion on this.

Reply:
Thanks for your input,
Yes, there is no default theme. The `cyverseTheme` is required to be passed to `MuiThemeProvider` when instantiating the app. 

So from this and my conversation with Jason looks like we will want to "fail fast" with a message explaining this.

By "override" I meant override these values at instantiation with an obj deep merge, or copy  `cyverseTheme` and change values manually.